core/src/test/java/io/confluent/kafka/schemaregistry/ClusterTestHarness.java

```diff
@@ -144,14 +133,33 @@ public void setUp() throws Exception {
 
     brokerList =
         TestUtils.getBrokerListStrFromServers(JavaConversions.asScalaBuffer(servers),
-                                              SecurityProtocol.PLAINTEXT);
+                                              getSecurityProtocol());
 
     if (setupRestApp) {
       restApp = new RestApp(choosePort(), zkConnect, KAFKASTORE_TOPIC, compatibilityType);
       restApp.start();
     }
   }
 
+  protected KafkaConfig getKafkaConfig(int brokerId) {
+    final Option<java.io.File> noFile = scala.Option.apply(null);
+    final Option<SecurityProtocol> noInterBrokerSecurityProtocol = scala.Option.apply(null);
+    Properties props = TestUtils.createBrokerConfig(
+            brokerId, zkConnect, false, false, TestUtils.RandomPort(), noInterBrokerSecurityProtocol,
+            noFile, true, false, TestUtils.RandomPort(), false, TestUtils.RandomPort(), false,
+            TestUtils.RandomPort(), Option.<String>empty());
+    props.setProperty("auto.create.topics.enable", "true");
+    props.setProperty("num.partitions", "1");
+    // We *must* override this to use the port we allocated (Kafka currently allocates one port
```

Comment:
Sorry, just notice that this was pasted from the original code. Still might be worth clarifying this comment.